bundler/spec/dependabot/bundler/file_updater_spec.rb

```diff
@@ -118,7 +118,7 @@
             .to include("\"statesman\", \"~> 1.2.0\"")
         end
 
-        context "for a gems.rb setup" do
+        context "when for a gems.rb setup" do
```

Comment:
Let's please try to make these sentences flow nicely, the idea of these context blocks is that, stringed together they explain in English what the test is trying to address, and those sentences should flow well, it makes it easier to understand the tests.

So in this case: `when using a gems.rb setup` for example, that way the entire test will read:

> Dependabot::Bundler::FileUpdater##updated_dependency_files the updated gemfile when using a gems.rb setup, it delegates to the GemfileUpdater